src/config.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = "http://localhost:8000";
```

Comment:
Aware this is bad practice, might pull out to an env var later but leaving here while in progress